frappe/public/scss/desk/dark.scss

```diff
@@ -93,6 +93,10 @@
 
 	--shadow-base:  0px 4px 8px rgba(114, 176, 233, 0.06), 0px 0px 4px rgba(112, 172, 228, 0.12);
 
+    // "diff" colors
+    --diff-added: var(--green-800);
+    --diff-removed: var(--red-800);
```

Comment:
Why this was not caught in Linter🤔

Reply:
I don't get it, what's not working here? 🤔

Reply:
Ah, resolved